src/github/nag.rs

```diff
@@ -185,7 +130,147 @@ fn update_proposal_review_status(proposal_id: i32) -> DashResult<()> {
     Ok(())
 }
 
+fn update_poll_response_status(poll_id: i32) -> DashResult<()> {
+    let conn = &*DB_POOL.get()?;
+    // this is an updated comment from the bot itself
+
+    // parse out each "responded" status for each user, then update them
+
+    let survey: Poll = poll::table.find(poll_id).first(conn)?;
+
+    // don't update any statuses if the poll is closed
+    if survey.poll_closed {
+        return Ok(());
+    }
+
+    let comment: IssueComment = issuecomment::table
+        .find(survey.fk_bot_tracking_comment)
+        .first(conn)?;
+
+    // parse the status comment and mark any new responses as responded
+    for username in parse_ticky_boxes("poll", survey.id, &comment) {
+        let user: GitHubUser = githubuser::table
+            .filter(githubuser::login.eq(username))
+            .first(conn)?;
```

Comment:
IIUC this will currently fail the entire poll if someone typo's one of the provided usernames. Do we have an error reporting mechanism for that?